react_app/src/Clock.js

```diff
@@ -0,0 +1,64 @@
+import React,{Component} from 'react';
+
+const clockTile = {
+  color: 'white',
+  textAlign:'center'
+}
+const clockSpan = {
+  padding : '4px'
+}
+let runner;
+class Clock extends Component{
+  constructor(props) {
+    super(props);
+    this.state ={
+      'time' : this.getCurrentTime()
+    }
+  }
+  getCurrentTime = () =>{
+    const locale = this.props.locale ? this.props.locale : [];
+    const hour12 = (this.props.hour12 === false) ? false : true;
```

Comment:
consider re-writing this as it is a little confusing. If hour 12 is false return false else true. Couldn't you just use the hour 12 variable without this ternary?